diff -ur emirror-2.1.17/Makefile.in emirror-2.1.17.php/Makefile.in
--- emirror-2.1.17/Makefile.in	Sat Sep  9 21:39:35 2000
+++ emirror-2.1.17.php/Makefile.in	Sat Sep  9 21:39:27 2000
@@ -6,6 +6,7 @@
 libdir=$(prefix)/lib/emirror
 etcdir=@sysconfdir@/emirror
 mandir=@mandir@/man1
+htmldir=@HTMLDIR@
 
 all: 
 	@echo There\'s no need to compile anything - use \"make install\"
@@ -26,7 +27,8 @@
 	@$(INSTALL) -m 644 etc/*.cfg $(DESTDIR)$(etcdir)
 	@$(INSTALL) -m 644 etc/email-addresses $(DESTDIR)$(etcdir)
 	@$(INSTALL) -m 644 etc/*.template $(DESTDIR)$(etcdir)
-	@$(INSTALL) -m 644 etc/*.gif $(DESTDIR)$(etcdir)
+	@$(INSTALL) -m 644 etc/*.gif $(DESTDIR)$(htmldir)
+	@$(INSTALL) -m 644 etc/*.inc $(DESTDIR)$(htmldir)
 	@$(INSTALL) -d $(DESTDIR)$(etcdir)/active
 	@$(INSTALL) -d $(DESTDIR)$(etcdir)/unused
 	@$(INSTALL) -m 644 etc/unused/*.cfg $(DESTDIR)$(etcdir)/unused
diff -ur emirror-2.1.17/etc/eclipt-index-php.template emirror-2.1.17.php/etc/eclipt-index-php.template
--- emirror-2.1.17/etc/eclipt-index-php.template	Sun Aug 20 15:19:47 2000
+++ emirror-2.1.17.php/etc/eclipt-index-php.template	Sat Sep  9 21:37:47 2000
@@ -1,4 +1,4 @@
-<? include "/home/htdocs/include/generic.inc"; ?>
+<? include "generic.inc"; ?>
 <? HtmlHeader(""); ?>
 
     <spacer type="vertical" size=15>
diff -ur emirror-2.1.17/etc/eclipt-logfile-php.template emirror-2.1.17.php/etc/eclipt-logfile-php.template
--- emirror-2.1.17/etc/eclipt-logfile-php.template	Sun Aug 20 17:10:07 2000
+++ emirror-2.1.17.php/etc/eclipt-logfile-php.template	Sat Sep  9 21:37:54 2000
@@ -1,4 +1,4 @@
-<? include "/home/htdocs/include/generic.inc"; ?>
+<? include "generic.inc"; ?>
 <? HtmlHeader(""); ?>
 
   <spacer type="vertical" size=15>
diff -ur emirror-2.1.17/etc/eclipt-summary-php.template emirror-2.1.17.php/etc/eclipt-summary-php.template
--- emirror-2.1.17/etc/eclipt-summary-php.template	Sun Aug 20 17:09:07 2000
+++ emirror-2.1.17.php/etc/eclipt-summary-php.template	Sat Sep  9 21:38:00 2000
@@ -1,4 +1,4 @@
-<? include "/home/htdocs/include/generic.inc"; ?>
+<? include "generic.inc"; ?>
 <? HtmlHeader(""); ?>
 
     <spacer type="vertical" size=15>
