diff -urN emirror-2.1.19/Makefile.in emirror-2.1.19.new/Makefile.in
--- emirror-2.1.19/Makefile.in	Sun Jan 21 20:10:27 2001
+++ emirror-2.1.19.new/Makefile.in	Thu Aug 16 21:21:05 2001
@@ -1,18 +1,19 @@
 
 prefix=@prefix@
 sysconfdir=@sysconfdir@
-bindir=$(prefix)/bin
+bindir=$(DESTDIR)$(prefix)/bin
 INSTALL=@INSTALL@
 PYTHON=@PYTHON@
-libdir=$(prefix)/lib/emirror
-etcdir=$(sysconfdir)/emirror
-mandir=$(prefix)/man/man1
+libdir=$(DESTDIR)$(prefix)/lib/emirror
+etcdir=$(DESTDIR)$(sysconfdir)/emirror
+mandir=$(DESTDIR)$(prefix)/share/man/man1
+htmldir=$(DESTDIR)@HTMLDIR@
 
 all: 
 	@echo There\'s no need to compile anything - use \"make install\"
 
 install :
-	@echo Installing files in $(prefix) ...
+	@echo Installing files in $(DESTDIR)$(prefix) ...
 	@$(INSTALL) -d $(bindir)
 	@$(INSTALL) emirror $(bindir)
 	@$(INSTALL) emirrortest $(bindir)
@@ -29,7 +30,8 @@
 	@$(INSTALL) -b -m 644 etc/*.cfg $(etcdir)
 	@$(INSTALL) -b -m 644 etc/email-addresses $(etcdir)
 	@$(INSTALL) -m 644 etc/*.template $(etcdir)
-	@$(INSTALL) -m 644 etc/*.gif $(etcdir)
+	@$(INSTALL) -m 644 etc/*.gif $(htmldir)
+	@$(INSTALL) -m 644 etc/*.inc $(htmldir)
 	@$(INSTALL) -d $(etcdir)/active
 	@$(INSTALL) -d $(etcdir)/unused
 	@$(INSTALL) -m 644 etc/unused/*.cfg $(etcdir)/unused
diff -urN emirror-2.1.19/bin/mirror.in emirror-2.1.19.new/bin/mirror.in
--- emirror-2.1.19/bin/mirror.in	Sun Dec  3 10:59:25 2000
+++ emirror-2.1.19.new/bin/mirror.in	Thu Aug 16 21:17:22 2001
@@ -17,4 +17,4 @@
 
 # Start the mirror program and update the index page 
 
-emirror -v -c @sysconfdir@/emirror/general.cfg -d @sysconfdir@/emirror/active/ -n @HTMLDIR@
+emirror -- -v -c @sysconfdir@/emirror/general.cfg -d @sysconfdir@/emirror/active/
diff -urN emirror-2.1.19/bin/updateindex.in emirror-2.1.19.new/bin/updateindex.in
--- emirror-2.1.19/bin/updateindex.in	Sun Dec  3 11:34:33 2000
+++ emirror-2.1.19.new/bin/updateindex.in	Thu Aug 16 21:17:33 2001
@@ -10,4 +10,4 @@
 
 # Start the mirror program and update the index page 
 
-emirror -v -c @sysconfdir@/emirror/general.cfg -n @HTMLDIR@
+emirror -- -v -c @sysconfdir@/emirror/general.cfg
diff -urN emirror-2.1.19/etc/eclipt-index-php.template emirror-2.1.19.new/etc/eclipt-index-php.template
--- emirror-2.1.19/etc/eclipt-index-php.template	Sun Apr 15 10:09:01 2001
+++ emirror-2.1.19.new/etc/eclipt-index-php.template	Thu Aug 16 21:21:34 2001
@@ -1,4 +1,4 @@
-<? include "/home/htdocs/include/generic.inc"; ?>
+<? include "generic.inc"; ?>
 <? HtmlHeader(""); ?>
 
     <spacer type="vertical" size=15>
diff -urN emirror-2.1.19/etc/eclipt-logfile-php.template emirror-2.1.19.new/etc/eclipt-logfile-php.template
--- emirror-2.1.19/etc/eclipt-logfile-php.template	Sun Apr 15 10:10:19 2001
+++ emirror-2.1.19.new/etc/eclipt-logfile-php.template	Thu Aug 16 21:21:18 2001
@@ -1,4 +1,4 @@
-<? include "/home/htdocs/include/generic.inc"; ?>
+<? include "generic.inc"; ?>
 <? HtmlHeader(""); ?>
 
   <spacer type="vertical" size=15>
diff -urN emirror-2.1.19/etc/eclipt-summary-php.template emirror-2.1.19.new/etc/eclipt-summary-php.template
--- emirror-2.1.19/etc/eclipt-summary-php.template	Sun Apr 15 10:10:48 2001
+++ emirror-2.1.19.new/etc/eclipt-summary-php.template	Thu Aug 16 21:21:47 2001
@@ -1,4 +1,4 @@
-<? include "/home/htdocs/include/generic.inc"; ?>
+<? include "generic.inc"; ?>
 <? HtmlHeader(""); ?>
 
     <spacer type="vertical" size=15>
diff -urN emirror-2.1.19/etc/general.cfg.in emirror-2.1.19.new/etc/general.cfg.in
--- emirror-2.1.19/etc/general.cfg.in	Fri Apr 13 09:21:43 2001
+++ emirror-2.1.19.new/etc/general.cfg.in	Thu Aug 16 21:18:59 2001
@@ -1,18 +1,21 @@
 [general]
+Download-url: ftp://localhost/pub/mirrors
+Parent: /home/ftp/pub/mirrors
 Parent-Format: None
 Output-directory: @HTMLDIR@
 Output-format: HTML
 Tries: 5
 Wait: 30
-Email: root@localhost
+Email: ftp@localhost
 Max-parallel: 3
 Recursive: Yes
+Follow-Symlinks: Yes
 Username: anonymous
-Password: anonymous@localhost
-Error-MailCmd: mail -s "EMirror Error Report" Martin.Preishuber@eclipt.at
+Password: emirror@localhost
+Error-MailCmd: mail -s "EMirror Error Report" ftp@localhost
 Mail-Users: @SYSCONFDIR@/emirror/email-addresses
 Mail-User-Cmd: mail -s "EMirror EMail-Information"
 Log-Template: @SYSCONFDIR@/emirror/logfile.template
 Index-Template: @SYSCONFDIR@/emirror/index.template
 Index-Directory: @HTMLDIR@
-Summary-Template: @SYSCONFDIR@/emirror/summary.template
\ No newline at end of file
+Summary-Template: @SYSCONFDIR@/emirror/summary.template
diff -urN emirror-2.1.19/man/emirror.1 emirror-2.1.19.new/man/emirror.1
--- emirror-2.1.19/man/emirror.1	Fri Apr 13 10:28:46 2001
+++ emirror-2.1.19.new/man/emirror.1	Thu Aug 16 21:19:55 2001
@@ -120,7 +120,7 @@
 .IP "--mail-users"
 Specifies a file, which contains a list of valid email-addresses
 of users, who want to get information about updates
-(e.g. /usr/local/etc/email-addresses)
+(e.g. /etc/emirror/email-addresses)
 .IP "--info-mailcmd"
 Command to send mail to users when one spcific mirror has been updated
 (e.g. mail -s "EMirror - Package updated: %s" user1@host1 user2@host2)
@@ -480,11 +480,11 @@
 .IR EMIRRORCFGPATH
 
 .SH FILES
-\fI/usr/local/lib/emirror/*\fP, 
-\fI/usr/local/etc/general.cfg\fP, 
-\fI/usr/local/etc/email-addresses\fP, 
-\fI/usr/local/etc/active/*\fP, 
-\fI/usr/local/etc/unused/*\fP
+\fI/usr/lib/emirror/*\fP, 
+\fI/etc/emirror/general.cfg\fP, 
+\fI/etc/emirror/email-addresses\fP, 
+\fI/etc/emirror/active/*\fP, 
+\fI/etc/emirror/unused/*\fP
 
 .SH UNRESTRICTIONS
 .PP
