--- emirror-2.1.17/configure.in~	Tue Aug 15 14:19:54 2000
+++ emirror-2.1.17/configure.in	Fri Apr 13 04:04:40 2001
@@ -34,7 +34,7 @@
 then
     echo failed. Get a more recent version from www.python.org !
     exit 1
-else
+elif test  $PYTHONMAJOR -lt 2 ; then
     if test $PYTHONMINOR -lt 5
     then
         echo failed. Get a more recent version from www.python.org !
@@ -42,6 +42,8 @@
     else
         echo ok
     fi
+else
+   echo ok
 fi
 
 dnl Check for ftplib
diff -ur emirror-2.1.17.orig/ftplibbeta.py emirror-2.1.17/ftplibbeta.py
--- emirror-2.1.17.orig/ftplibbeta.py	Tue Aug 15 10:43:32 2000
+++ emirror-2.1.17/ftplibbeta.py	Mon Apr 16 02:16:18 2001
@@ -114,7 +114,7 @@
 		if port: self.port = port
 		self.passiveserver = 0
 		self.sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-		self.sock.connect(self.host, self.port)
+		self.sock.connect((self.host, self.port))
 		self.file = self.sock.makefile('rb')
 		self.welcome = self.getresp()
 		return self.welcome
@@ -264,7 +264,7 @@
 		if self.passiveserver:
 			host, port = parse227(self.sendcmd('PASV'))
 			conn = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-			conn.connect(host, port)
+			conn.connect((host, port))
 			if (rest != None):
 				self.sendcmd("REST %.0f" % rest)
 			resp = self.sendcmd(cmd)
